src/open_inwoner/js/components/questionnaire/index.js

```diff
@@ -0,0 +1,10 @@
+const question = document.getElementById('question')
+const stepIndicators = document.querySelectorAll(
```

Comment:
There should only be one active item. So I would say that we should not use querySelectorAll.
Also by selecting multiple (if there are) you will scroll to the first, than the second, than the third.